members/CusRc79444e91674069572.yaml

```diff
@@ -0,0 +1,104 @@
+bioguide: CusRc79444e91674069572
+contact_form:
+  driver: chrome
+  steps:
+    - visit: "https://parkplanning.nps.gov/commentForm.cfm?documentID=125324"
+    - fill_in:
+        - name: city
+          selector: "#id_authorCity"
+          value: $ADDRESS_CITY
+          required: true
+        - name: zipcode
+          selector: "#id_authorPostalCode"
+          value: $ADDRESS_ZIP5
+          required: true
+        - name: first name
+          selector: "#id_authorFirstName"
+          value: $NAME_FIRST
+          required: false
+        - name: last name
+          selector: "#id_authorLastName"
+          value: $NAME_LAST
+          required: false
+        - name: address 1
+          selector: "#id_authorAddress1"
+          value: $ADDRESS_STREET
+          required: false
+        - name: address 2
+          selector: "#id_authorAddress2"
+          value: $ADDRESS_STREET_2
+        - name: email
+          selector: "#id_authorEmail"
+          value: $EMAIL
+          required: false
+        - name: message
+          selector: "#id_pubComments"
+          value: $MESSAGE
+          required: true
+    - select:
+        - name: state
+          selector: "#id_authorStateProvDD"
+          value: ADDRESS_STATE_FULL
+          required: true
+          options:
+            AL: Alaska
+            AK: Alabama
+            AS: American Samoa
+            AZ: Arizona
+            AR: Arkansas
+            CA: California
+            CO: Colorado
```

Comment:
Does this list need to match up exactly with the options in the State/Territory dropdown on the form?  If so - I noticed "Commonwealth of the Northern Mariana Islands" was missing, but that might have been intentional.  Also - is the assumption here the supporter is US-based so we don't bother implementing the country dropdown (I see it's defaulted to US)?  Otherwise, this PR looks good